netlink/net_ns.go

```diff
@@ -0,0 +1,28 @@
+//+build linux
+
+package netlink
+
+import (
+	"os"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// guessRootNetNSFd guesses the file descriptor of the root net NS
+// and returns 0 in case of failure
+func guessRootNetNSFd(procRoot string) int {
+	for _, path := range []string{
+		procRoot + "/1/ns/net",
```

Comment:
`procRoot` will return either the `/host/proc` or `/proc`. Why do we need to try both?

Reply:
I suppose we don't. This is a holdover from when this was doing more of a guess